### PR TITLE
[AuditLogging] add the help link for storage location

### PR DESCRIPTION
**Pull request by tianleh**
*Issue #, if available:*

*Description of changes:*
Add the help link to the target documentation link.

*Test*:
Verified that the doc is opened in a separate tab.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/panels/audit-logging/audit-logging.tsx [hunk 1]
```diff
@@ -64,8 +64,8 @@ function renderStatusPanel(onSwitchChange: () => void, auditLoggingEnabled: bool
               <EuiCode>elasticsearch.yml</EuiCode>. The default storage location is{' '}
               <EuiCode>internal_elasticsearch</EuiCode>, which stores the logs in an index on this
               cluster.{' '}
-              <EuiLink external={true} href="/">
-                Learn more
+              <EuiLink external={true} href={DocLinks.AuditLogsStorageDoc} target="_blank">
+                Learn More
```
Comment (yuxitang-amzn) [inline public/apps/configuration/panels/audit-logging/audit-logging.tsx:68]: Is it intended?

Comment (tianleh) [reply public/apps/configuration/panels/audit-logging/audit-logging.tsx:68]: Copied from landing page. After checking the code base, it has mixed "Learn More" and "Learn more". But the mockup is using "Learn more". Let me do a bulk fix.

File: public/apps/configuration/panels/audit-logging/audit-logging.tsx [hunk 2]
```diff
@@ -64,8 +64,8 @@ function renderStatusPanel(onSwitchChange: () => void, auditLoggingEnabled: bool
               <EuiCode>elasticsearch.yml</EuiCode>. The default storage location is{' '}
               <EuiCode>internal_elasticsearch</EuiCode>, which stores the logs in an index on this
               cluster.{' '}
-              <EuiLink external={true} href="/">
-                Learn more
+              <EuiLink external={true} href={DocLinks.AuditLogsStorageDoc} target="_blank">
```
Comment (yuxitang-amzn) [inline public/apps/configuration/panels/audit-logging/audit-logging.tsx:67]: Non-blocking, I would suggest have an `ExternalLink` component that accepts just an href prop.

**Review by yuxitang-amzn — APPROVED** [review]

File: public/apps/configuration/panels/audit-logging/audit-logging.tsx (hunk 2 above)
Comment (tianleh) [reply public/apps/configuration/panels/audit-logging/audit-logging.tsx:67]: I will add a TODO

**Pull request merged**